packages/rx_bloc_list/CHANGELOG.md

```diff
@@ -1,7 +1,13 @@
+## [3.2.0-dev.2] June 7, 2022
+- Added `IdentifiablePair<E extends Identifiable>` class
+- Added `Stream<IdentifiablePair<E extends Identifiable>>` extensions: `withLatestFromIdentifiablePairList`
+- Removed parameter `identifiableInList` from `operationCallback` in `Stream<T extends Identifiable>` extension: `withLatestFromIdentifiableList`
+
 ## [3.2.0-dev.1] January 30, 2022
 - Added `Identifiable` interface for handling inter-feature communication
 - Added `List<T extends Identifiable>` extensions: `ids`, `containsIdentifiable`, `removeIdentifiable`, `mergeWith`
-- Added `Stream<T extends Identifiable>` extensions: `withLatestFromIdentifiableList`
+- Added `Stream<T extends Identifiable>` extensions: `mapCreatedWithLatestFrom`, `mapUpdatedWithLatestFrom`, `mapDeletedWithLatestFrom`, `withLatestFromIdentifiableList`
```

Comment:
3.2.0-dev.1 should remain unchanged, please add all new functionalities/fixes in the [3.2.0-dev.2] changelog